src/audio/mixer/mixer.c

```diff
@@ -119,33 +121,39 @@ static int mixer_process(struct processing_module *mod,
 		if (!audio_stream_set_zero(mod->output_buffers[0].data, sink_bytes))
 			mod->output_buffers[0].size = sink_bytes;
 
-		md->sources_inactive = true;
-
```

Comment:
TODO: Since this field is no longer used, should I try to remove it?

Reply:
yes, remove it if no longer used.